addons/binding/org.openhab.binding.yamahareceiver/README.md

```diff
@@ -23,6 +23,26 @@ Bridge yamahareceiver:yamahaAV:ReceiverID "Yamaha Receiver Bridge Name" [host="a
 If your receiver is using menu-based net radio navigation, you can use this binding to
 select radio stations from a configured menu.
 
+Configuration parameters for Bridge type `yamahareceiver`:
+
+Parameter | Required | Default | Description
+--------|------|------|------
+`host` | yes | / | The IP address of the AVR to control
+`port` | no | 80 | The API port of the AVR to control
+`refreshInterval` | no | 60 | Refresh interval in seconds
+`albumUrl` | no | / | When the album image is not provided by the Yamaha input source, you can specify the default image URL to apply
+`inputMapping` | no | / | Some Yamaha models return different input values on status update than required in the change input commands. See [below](#input-values) for details
```

Comment:
The default is actually an empty string, so I suggest to remove the /

Reply:
I was not sure how to point the "not-a-programmer" user that the default is an empty string. So I added "" and in addition a hint that this means an empty string. WDYT?

Reply:
Would be nice to have a specific example here, as I can't see how this is exactly configured (even in the details)